esp/src/eclwatch/templates/ECLSourceWidget.html

```diff
@@ -1,6 +1,6 @@
 <div class="${baseClass}">
     <div id="${id}BorderContainer" class="${baseClass}BorderContainer" style="width: 100%; height: 100%; overflow: hidden" data-dojo-props="splitter: false, gutters:false" data-dojo-type="dijit.layout.BorderContainer">
-        <div id="${id}EclContent" class="centerPanel" data-dojo-props="region: 'center'" data-dojo-type="dijit.layout.ContentPane">
+        <div id="${id}EclContent" class="centerPanel" style="padding:0px" data-dojo-props="region: 'center'" data-dojo-type="dijit.layout.ContentPane">
```

Comment:
; is a statement separator and not a statement terminator - which is why this is ok.